code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -231,13 +250,12 @@
 

```

Comment:
I have tested it and it only locks I don't see what did I change that could make it happen like that so I think someone else made it like that either by mistake or on purpose, whichever reason will affect if I change it or not.